Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -1092,10 +1151,14 @@ protected function reflectClass(string $className): void
         }
 
         $class = new ClassReflection($className);
-        if (!isset($this->classReflectionData[$className])) {
+        if (!isset($this->classReflectionData[$className]) || !is_array($this->classReflectionData[$className])) {
             $this->classReflectionData[$className] = [];
         }
 
+        if (!isset($this->classReflectionData[$className][self::DATA_INTERFACE_IMPLEMENTATIONS]) && $class->isInterface()) {
+            $this->classReflectionData[$className][self::DATA_INTERFACE_IMPLEMENTATIONS] = [];
+        }
```

Comment:
we dont need that explicitly here but it doesnt hurt either right? should be done automatically in `addImplementedInterface`